spec/models/book_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+require 'faker'
+
+describe Book do
+  subject(:book) do
+    Book.new(
```

Comment:
Use the factory you created: `build(:book)`

Reply:
Done, now books are create with factory_bot.